### ISSUE TITLE
Waiting time between retries

### ISSUE THREAD

**Issue by jonathansp** [muicss/loadjs#71]
`numRetries` is a good approach to get around to mobile network issues. 

Retrying a couple of time in a row does not mean that loadjs will load successfully. Maybe waiting for some seconds before trying again would be better.

Could you please add an optional parameter in order to have loadjs wait some seconds between retries?

**Comment by amorey** [comment]
Can you describe the scenario you're running into in a bit more detail? Is the purpose of a timer to deal with connectivity issues?

**Comment by azizhk** [comment]
Yup the purpose of the timer is to space out between requests hoping that the internet comes back on.

Something like:
```js
{
  numRetries: 5,
  retryWaitTime: 1000 // 1 second
}
```
